cmd/supernode/app/gen_doc.go

```diff
@@ -36,7 +36,7 @@ func init() {
 	genDocCommand := &GenDocCommand{}
 	genDocCommand.cmd = &cobra.Command{
 		Use:           "gen-doc",
-		Short:         "Generate Document for dfget command line tool with MarkDown format",
+		Short:         "Generate Document for supernode command line tool with MarkDown format",
```

Comment:
` with MarkDown format`  I think `with` isn't  appropriate,`in` can be better?

Reply:
Agree